job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java

```diff
@@ -1,12 +1,12 @@
-package spark.jobserver;
+// package spark.jobserver;
```

Comment:
I don't think spark.jobserver is a valid folder for java, you need spark/jobserver.

Did you try that?